torchtune/utils/_checkpointing/_checkpointer_utils.py

```diff
@@ -11,14 +11,17 @@
 import torch
 import torch.nn as nn
 import torch.optim as optim
+from safetensors import safe_open
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
 
 from torchtune.utils._distributed import contains_fsdp
+from transformers.utils import is_safetensors_available
```

Comment:
I think we shouldn't import from transformers here as it's not in our core dependencies. If you've added safetensors to our core dependencies (based on the above comment) probably don't need to do this check anyways.

Reply:
yeah, I remove it in the latest version.